docs/source/rbac.rst

```diff
@@ -292,108 +295,197 @@ For example:
 
 .. _rbac-using_rbac:
 
+Automatically granting roles based on the user LDAP group membership
+--------------------------------------------------------------------
+
+.. note::
+
+   This functionality is only available in |st2| v2.3.0 and above when LDAP auth backend is used
+   for authentication.
+
+In addition to manually assigning roles to the users based on the definitions in the
+``/opt/stackstorm/rbac/assignments/`` directory, |st2| also supports automatically granting roles
+to users on authentication based on the LDAP groups membership.
+
+This comes handy in enterprise environments and makes |st2| user provisioning easier and faster.
+It means operator doesn't need to manually write and manage RBAC role assignment files on disk,
+because roles are automatically granted to the users based on their LDAP group membership and
+mappings files in ``/opt/stackstorm/rbac/mappings/`` directory.
+
+To be able to utilize this feature it first needs to be enabled in ``st2.conf`` by setting
+``rbac.sync_remote_groups`` option to ``True``.
+
+.. code-block:: ini
+
+  [rbac]
+  sync_remote_groups = True
+
+After this feature is enabled, operator needs to write mapping files which tell |st2| which roles
+to automatically grant to users which are a member of a particular LDAP group. Mapping files are
```

Comment:
s/...to users which are a member of.../...to users who are members of.../

Or maybe re-word the whole clause as something like "After this feature is enabled, the StackStorm administrator needs to write mapping files that tell |st2| which roles to automatically grant to users, based upon LDAP group membership"